app/src/main/java/com/github/polypoly/app/base/game/Transaction.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.polypoly.app.base.game
+
+import com.github.polypoly.app.base.menu.lobby.GameMode
+
+/**
+ * Represent the different transactions that can happen in a round
+ */
+enum class Transaction(private val title : String, private val duringRound: Boolean, private val gameModes : List<GameMode>) {
+    TAX_TO_PAY ("Taxes to pay:", false, listOf(GameMode.RICHEST_PLAYER, GameMode.LAST_STANDING)),
+    TAX_TO_RECEIVE("You taxed players: ", false, listOf(GameMode.RICHEST_PLAYER, GameMode.LAST_STANDING)),
+    MILESTONES_INCOME_TO_RECEIVE("Milestones reward: ", false, GameMode.values().toList()),
+    PASSIVE_LANDLORD_TAX_PAID("Passive taxes to pay: ", true, listOf(GameMode.LANDLORD)),
+    PASSIVE_LANDLORD_TAX_RECEIVED("You taxed passively players: ", true, listOf(GameMode.LANDLORD)),
+    TRADE_MONEY_PAID("Spent in trades: ", true, GameMode.values().toList()),
+    TRADE_MONEY_RECEIVED("Earned in trades: ", true, GameMode.values().toList()),
+    BUILDING_BOUGHT_PAID("Spent in buildings: ", true, GameMode.values().toList());
```

Comment:
The messages don't each reveal the same intent: for example, `Taxes to pay:` tells that the taxes are not payed yet, while `You taxed players:` tells that the taxes are already payed, is it intended?

Reply:
Moreover, this kind of strings are not really titles but descriptions that should be moved to the UI instead.

Reply:
you are right. I will make these more clear. Also I will change them into real titles, and have the formatting part in the toString function, as it is a much more logical way to go.